node/libs/crypto/src/bn254/mod.rs

```diff
@@ -0,0 +1,213 @@
+//! BLS signature scheme for the BN254 curve.
+
+use crate::ByteFmt;
+use anyhow::anyhow;
+use ark_bn254::{Bn254, Fr, G1Projective as G1, G2Projective as G2};
+use ark_ec::{
+    pairing::{Pairing as _, PairingOutput},
+    Group as _,
+};
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+pub use error::Error;
+use num_traits::Zero as _;
+use std::{collections::HashMap, fmt::Debug, hash::Hasher};
```

Comment:
`fmt::Debug` import is unused (Clippy doesn't complain because it's used as a derive macro, but it's imported as a derive macro anyway).

Reply:
Fixed.